src/Netsells/SSOClient/UserProvider.php

```diff
@@ -17,7 +19,7 @@ class UserProvider implements UserProviderContract
      */
     public function retrieveById($identifier)
     {
-        return $this->getSSOUser(['id' => $identifier]);
+        return;
```

Comment:
Return type should be null rather than void. I know that implementation wise it's the same but my editor is sad 😢